### ISSUE TITLE
Audio dropout with AAC Codec Omega 21.1 

### ISSUE THREAD

**Issue by Simkin84** [xbmc/xbmc#25665]
### Describe the bug
Here is a clear and concise description of what the problem is:

After upgrading to 21.1 from 21.0 i experience audio dropouts on video files with AAC codec.

It does also seem to happen on other codecs like Dolby Digital but not that often. Briefly tried DTS-HD and TrueHD/Atmos and they seemed fine.

The Speaker icons on my receiver will flicker when this happens, and i also experienced some big video stutter together with the audio drop out on a video file with AAC.

Did not have this issue on 21.0.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

No audio dropout and video stutter

## Actual Behavior

Audio dropouts (last for 1 sec each time approx)

### To Reproduce

1. Play video file with AAC codec
2.
3.

### Debuglog

The debuglog can be found here:

https://paste.kodi.tv/ajeropesim.kodi

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] macOS
 - [X] Windows 11 Home 23H2
 - [ ] Windows UWP
 - [X] AMD Ryzen 9 7900
 - [X] AMD Radeon RX 7600 XT
 - [X] Yamaha RX-A 3070 (Receiver)
 - [X] LG OLED CX 77 (TV)

 - Operating system version/name:
 - Kodi version: 21.1
 - Windows 11 Home 23H2

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by thexai** [comment]
There is no change from 21.0 to 21.1 that affects audio AAC specifically. It is also a format that is decoded by ffmpeg and played as PCM (regardless of whether AVR is used or not). This is also not passthrough related.

No one else has complained about this and you are not sure yourself according to: https://forum.kodi.tv/showthread.php?tid=378596&pid=3207820#pid3207820

Closing for the moment.

**Comment by mrfixit2001** [comment]
I went searching for others that reported a similar issue to what I'm experiencing and came across this issue. Just wanted to add to this that I also seem to be experiencing a random audio drop on 21.1, altho I cannot confirm it's *only* with AAC. It does not happen immediately after playback starts, it takes awhile - ranging from 15 mins to 40 mins to first begin to occur. I've noticed it more often when I have my audio config correctly set to 7.1, whereas using 2.0 it seems to happen less often and/or takes longer to begin to occur. Once it starts happening, it doesn't go away until I toggle some of the audio properties - such as switching the setup between 7.1 and 2.0, which I expect is resetting the audio system. It's fine for awhile after that but will eventually begin reoccurring.

The only relevant log entry is "OutputPicture - timeout waiting for buffer" - but I'm unsure at this time if this entry is before the issue occurs or after I've toggled an audio change to temporarily work around the issue. I'll see if I can get more detailed logs.

My setup is a Pine64 Quartz64-B RK3566 running a kodi compiled using the buildroot toolchain from tag kodi-21.1-Omega. It's connected to an AV receiver with a 7.1 setup. No settings on my AV receiver have any affect on this issue.

I can also confirm this did not happen in v21.0.
